corehq/apps/hqwebapp/static/hqwebapp/js/tempus_dominus.js

```diff
@@ -20,7 +20,7 @@ hqDefine("hqwebapp/js/tempus_dominus", [
     window.Popper = Popper;
 
     let createDatePicker = function (el, options) {
-        return new tempusDominus.TempusDominus(el, _addDefaultOptions(options, {
+        let picker = new tempusDominus.TempusDominus(el, _addDefaultOptions(options, {
```

Comment:
Is the return not needed anymore? I would have expected a `return picker` below.

Reply:
never mind. I see it now